tests/upgrade_part2.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+- hosts: all
+  tasks:
+    - name: grab old volume ID
+      command: ls /var/lib/docker/vfs/dir
+      register: vfs_dir_result
+    - name: store old volume ID
+      set_fact:
+        old_volume_id: "{{ vfs_dir_result.stdout }}"
+
+- hosts: all
+  roles:
+    - role: docker
+      docker_version: 1.11.2
```

Comment:
Our upgrade use case is specifically from 1.5.0 to 1.11.2. I'm being selfish. :)